src/common/interfaces.ts

```diff
@@ -114,10 +134,14 @@ export interface IUpdateMosaicRequest {
 }
 
 export interface IFileProvider {
```

Comment:
rename IFileProvider to IConfigAdapterProvider

Reply:
maybe IConfigProvider?